.gitlab-ci.yml

```diff
@@ -97,6 +141,30 @@ polkadot-js-ui-ink-examples:
     - WATERFALL_SKIP_CONTRACT_BUILD=true cargo test --jobs 4 --features headless,polkadot-js-ui
   after_script:
     - *shutdown-substrate-contracts-node
+  dependencies:
+    - parent-vars
+
+polkadot-js-ui-ink-examples-compare:
+  stage:                           compare-build
+  <<:                              *kubernetes-env
+
+  # comparison is made only if a parent (trgr) was created on prs. otherwise we would compare master with master
+  # TODO: while we lack comparison data for master, lets "compare" everything
+  # <<:                              *if-trgr-ref-is-master
+
+  # TODO: run on paritytech/ink-waterfall-ci, alpine is just for example here
+  variables:
+    CI_IMAGE: "alpine"
```

Comment:
Is there anything holding us back from swapping in the `ink-waterfall-ci` image here?

Reply:
Removed `alpine` override.